libs/components/tabs/testing/src/modules/tabs/tabset-harness.ts

```diff
@@ -163,6 +171,13 @@ export class SkyTabsetHarness extends SkyComponentHarness {
     return await this.locatorFor(SkyTabContentHarness.with({ tabId: id }))();
   }
 
+  /**
+   * Whether the tabset is a Wizard component.
+   */
+  public async isWizardTabset(): Promise<boolean> {
+    return (await (await this.host()).getAttribute('tabstyle')) === 'wizard';
```

Comment:
This is fragile because `tabStyle` is an input that can be bound to. We should add a `data-tab-style` HostBinding to the tabset component that we can query here: 

```
@HostBinding('attr.data-tab-style')
public get dataTabStyle(): string {
  return this.tabStyle;
}
```